src/actions/authActions.js

```diff
@@ -97,9 +106,17 @@ const getUserData = userArray => (dispatch) => {
   return dispatch(getUserDataFailure());
 };
 
+const logoutUser = () => async (dispatch) => {
+  window.localStorage.removeItem('token');
+  window.localStorage.removeItem('rememberToken');
```

Comment:
Hi @davidokonji How do you get rid of the encoded state in the localStorage?

Reply:
@Chiazokam there is an action which is used to restore the state to its initial state